packages/exceptions/index.ts

```diff
@@ -9,8 +9,8 @@ export * from './src/format/FormatException';
 export * from './src/format/VersionFormatException';
```

Comment:
Please make sure all version exceptions are exported and there is generic `VersionException`

Reply:
Not sure what are you referring to? There's only `VersionFormatException` in the repository and it is exported

Reply:
Ok